static/app/components/featureFeedback/index.tsx

```diff
@@ -0,0 +1,37 @@
+import {openModal} from 'sentry/actionCreators/modal';
+import Button from 'sentry/components/button';
+import {IconMegaphone} from 'sentry/icons';
+import {t} from 'sentry/locale';
+
+import {FeedBackModalProps} from './feedbackModal';
+
+interface Props extends FeedBackModalProps {}
+
+// Provides a button that, when clicked, opens a modal with a form that,
+// when filled and submitted, will send feedback to Sentry (feedbacks project).
+export function FeatureFeedback({feedbackTypes, featureName}: Props) {
```

Comment:
Should this be `FeatureFeedbackButton` then?

Reply:
I thought about that too, but the feature it's more than just a button... it opens a modal 🤷‍♀️

Reply:
actually, I like the idea 🙂 will update the code... thanks

Reply:
will leave this for you once you are back @priscilawebdev